app/src/main/res/drawable/my_recycleview_element_border.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
in my app, its what i use to give my objects a black border, ive renamed it thought :)